docs/example.md

```diff
@@ -2,7 +2,7 @@
 
 After following the [install instructions](install.md), you can deploy an example application with:
 
-    curl https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/v0.1.2/examples/color.yaml | kubectl apply -f -
+    curl https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/examples/color.yaml | kubectl apply -f -
```

Comment:
Just curious, why not just
```
kubectl apply -f https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/examples/color.yaml
```

Reply:
Probably cause I didn't know you could do that.